Examples/Framework/include/ActsExamples/EventData/MuonSimHit.hpp

```diff
@@ -61,7 +61,7 @@ Acts::GeometryIdentifier::Value compressId(muonMdtIdentifierFields f) {
   using longIdType = Acts::GeometryIdentifier::Value;
   longIdType out{0};
   out = out << g_fieldShift | f.stationName;
-  out = out << g_fieldShift | (u_int8_t)f.stationEta;
+  out = out << g_fieldShift | static_cast<u_int8_t>(f.stationEta);
```

Comment:
```suggestion
  out = out << g_fieldShift | static_cast<std::uint8_t>(f.stationEta);
```

Reply:
I removed the casts here completely, because I think we don't need them. `Acts::GeometryIdentifier::Value` is 64bit after all.